### DISCUSSION TITLE
Modified row spacing in Bootstrap 5

### DISCUSSION THREAD

**Discussion by SoftCircuits** [twbs/bootstrap#36385] [Q&A] [answered]
I have the following markup.

```
<div class="row">
    <div class="col-md-12">
        <div class="form-group">
            <label class="control-label" for="Truck_Notes">Notes</label>
            <textarea rows="3" class="form-control" id="Truck_Notes" name="Truck.Notes"></textarea>
            <span class="text-danger field-validation-valid" data-valmsg-for="Truck.Notes" data-valmsg-replace="true"></span>
        </div>
    </div>
</div>
<div class="row">
    <div class="col-md-8">
        <div class="form-group">
            <input type="submit" value="Submit" class="btn btn-primary">
            <a class="btn btn-secondary" href="/Trucks">Cancel</a>
        </div>
    </div>
    <div class="col-md-4">
        <div class="form-group text-end">
        </div>
    </div>
</div>
```

It looked fine in Bootstrap 4. But in Bootstrap 5 there is no spacing between the two rows.

![xsMLn](https://user-images.githubusercontent.com/3191331/168867627-14e876aa-1347-4957-99f5-612c2d030a66.png)

Bootstrap has always been good about spacing between rows. What is the preferred way in Bootstrap 5 to have appropriate spacing here?

**Comment by wnunezc** [comment]
you can fix it adding mb-2 to row element that contain text area.

https://jsfiddle.net/oL5trv31/

**Reply by SoftCircuits** [reply]
Thanks. Is requiring multiple classes to duplicate the layout we had in Bootstrap considered an improvement?

**Reply by wnunezc** [reply]
sorry i'm not a maintainer. maybe if you publish it as an issue or improvement, they will take it into account for version 5.2 that is about to come out.

**Comment by mdo** [comment, marked as answer]
`.form-group` was dropped in v5 because it was only a `margin-bottom` declaration. It didn't make much sense to have `.mb-3` and `.form-group` doing the same thing, so we went with the former utility class. It's also worth clarifying for other readers that `.row`s haven't had any vertical spacing by default in v4 or v5. This is strictly about `.form-group` no longer existing in v5.

**Reply by SoftCircuits** [reply]
Okay, thanks for explaining that in a way that finally makes sense to me.
